### PR TITLE
#3896 - Mask Institution User Income View

### PR DESCRIPTION
**Pull request by andrewsignori-aot**
- Masked the `totalFamilyIncome` following the same approach used to mask the MSFAA.
- Masked only for public institutions.
- The method used to format the money in the NOA UI when it is a number has no issues with the "masked" string.
image

### Sample UI with the masked value

image

### CODE REVIEW

File: sources/packages/backend/apps/api/src/route-controllers/assessment/assessment.controller.service.ts [hunk 1]
```diff
@@ -88,8 +94,13 @@ export class AssessmentControllerService {
       );
     }
 
+    const assessmentDTO = assessment.assessmentData as AssessmentAPIOutDTO;
```
Comment (dheepak-aot) [inline sources/packages/backend/apps/api/src/route-controllers/assessment/assessment.controller.service.ts:97]: we can do it like this without force casting. 

```ts
const assessmentDTO: AssessmentAPIOutDTO = assessment.assessmentData;
    if (maskTotalFamilyIncome) {
      assessmentDTO.totalFamilyIncome = MASKED_MONEY_AMOUNT;
    }
```

**Review by dheepak-aot — APPROVED** [review]
Looks good. One suggestion. 👍

File: sources/packages/backend/apps/api/src/route-controllers/assessment/assessment.controller.service.ts (hunk 1 above)
Comment (andrewsignori-aot) [reply sources/packages/backend/apps/api/src/route-controllers/assessment/assessment.controller.service.ts:97]: The cast is necessary to allow it to accept a string.
image

File: sources/packages/backend/apps/api/src/route-controllers/assessment/assessment.students.controller.ts [hunk 2]
```diff
@@ -66,6 +66,7 @@ export class AssessmentStudentsController extends BaseController {
     return this.assessmentControllerService.getAssessmentNOA(assessmentId, {
       studentId: userToken.studentId,
       maskMSFAA: false,
+      maskTotalFamilyIncome: false,
```
Comment (guru-aot) [inline sources/packages/backend/apps/api/src/route-controllers/assessment/assessment.students.controller.ts:69]: 👍

**Review by guru-aot — APPROVED** [review]
LGTM, nice work @andrewsignori-aot

File: sources/packages/backend/apps/api/src/route-controllers/assessment/assessment.controller.service.ts (hunk 1 above)
Comment (andrewsignori-aot) [reply sources/packages/backend/apps/api/src/route-controllers/assessment/assessment.controller.service.ts]: Changed as suggested.

**Pull request merged**